src/main.rs

```diff
@@ -392,6 +399,23 @@ async fn main_impl() -> Result<()> {
             }
             Ok(())
         }
+        Commands::Validate { rpc_port } => {
+            let client = make_rpc_client(rpc_port).await?;
+            let mut response = client.server_streaming(ValidateRequest).await?;
+            while let Some(item) = response.next().await {
+                let item = item?;
+                println!(
+                    "{} {} {} {}",
+                    item.path
+                        .map(|x| x.display().to_string())
+                        .unwrap_or("Collection".to_owned()),
+                    Blake3Cid(item.hash),
+                    HumanBytes(item.size),
+                    item.error.unwrap_or("OK".to_owned()),
```

Comment:
Would be nice to have like a checkmark and a cross for each state

Reply:
You got it...

![image](https://user-images.githubusercontent.com/248257/225903799-c9d26ff3-016c-4dda-a9fc-ac80f23402c0.png)